mandatory/3-playing-computer.js

```diff
@@ -6,31 +6,31 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
-  2. Remove the line that throws the error.
+  1. This program throws an error. Why? (If you can't find it, try executing it). missing b variable
+  2. Remove the line that throws the error. line 31
   3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
+  4. How many times is "f1" called? called 2 time (line 30,31)
+  5. How many times is "f2" called? called 3 times (line 29-31)
+  6. What value does the "a" parameter take in the first "f1" call? value 6 ? not sure
   7. What is the value of the "a" outer variable when "f1" is called for the first time?
 */
 
 let x = 2;
 let a = 6;
 
-const f1 = function (a, b) {
+const f1 = function(a, b) {
   return a + b;
 };
 
-const f2 = function (a, b) {
+const f2 = function(a, b) {
   return a + b + x;
 };
 
 console.log(x);
 console.log(a);
 console.log(b);
 
-for (let i = 0; i < 5; ++i) {
+for (let i = 0; i < 5; i++) {
```

Comment:
Here the question wants to tell you the difference between `++i` and `i++`. How this difference will change the output of this for loop?
We can talk about this in our next 1-1 and/or you can search Google to learn more about it :)